packages/client/tsconfig.json

```diff
@@ -1,4 +1,19 @@
 {
+  "compilerOptions": {
+    "strict": true,
+    "skipLibCheck": true,
+    "moduleResolution": "node",
+    "baseUrl": ".",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+
+    "noEmit": true
+  },
+  "references": [
```

Comment:
https://www.typescriptlang.org/docs/handbook/project-references.html

Reply:
Need to run this with the build flag: `tsc -b`

Reply:
this disable emission of files if there is an error. I guess this is ok to enable. Intresting that this solve the problem.